docs/source/_static/install_dependencies.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-trap 'RET=$? ; echo -e >&2 "\n\x1b[31mFailed installing dependencies. Could be a bug in the installer or unsupported platform. Open a bug report over at https://github.com/luxonis/depthai - exited with status $RET at line $LINENO \x1b[0m\n" ; exit $RET' ERR
```

Comment:
Reason for removing this?

Reply:
That must be a mistake. I have not deleted it on purpose for sure. Thank you for noticing

Reply:
corrected

Reply:
This very likely shouldn't be on a separate line. Did you test this @PetrNovota ?